Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3
+FROM blackdentech/kf-pandas
```

Comment:
Lol is this temporary? Also what's different with this image vs the standard Python 3 image

Reply:
bakes in pandas because that python:3 image takes a million years to compile pandas and then also breaks when Cython is missing